src/components/MenuBars/MenuItem/MenuButton.tsx

```diff
@@ -10,10 +11,29 @@ type MenuButtonProps = {
 };
 
 function MenuButton(props: MenuButtonProps) {
+  const [clickCount, setClickCount] = useState(0);
   const Icon = props.icon;
 
+  useEffect(() => {
+    if (clickCount > 0) {
```

Comment:
Works probably not as expected 🤔

Reply:
Yes, that's not the intended behaviour and I've already located the issue. The problem is that currently I prevent the default behaviour in the `touchstart` event so that the `click` event doesn't get fired. But this also prevents the normal event bubbling and therefore a touch event never reaches the window event listener. 

I have to think about a possible workaround / alternative.